### PR TITLE
Update translations for ARR and HW

### PR DESCRIPTION
**Pull request by Akurosia**
lot of small fixes in whole ARR and HW.
This PR should remove all FIXME entries as well as fix some translations.
Also "'missingTranslations': true," should be added where needed (hopefully i did not miss any)

### CODE REVIEW

File: ui/raidboss/data/02-arr/raid/t12.js [hunk 1]
```diff
@@ -159,111 +159,128 @@
...
       'replaceSync': {
-        'Phoenix-Egi': 'Phénix-Egi',
+        'Bennu': 'bénou',
         'Phoenix(?!-)': 'Phénix',
+        'Phoenix-Egi': 'Phénix-Egi',
       },
       'replaceText': {
-        'Bennu Add': 'Add Bénou',
+        '(?<! )Rebirth': 'Résurrection',
+        'Bennu Add': 'Bénou Add',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t12.js:182]: line 182 : In french is: Add Bénou thank you

File: ui/raidboss/data/02-arr/raid/t12.js [hunk 2]
```diff
@@ -159,111 +159,128 @@
...
+      '~effectNames': {
+        'Chain Of Purgatory': 'Kette der Purgation',
+      },
     },
     {
       'locale': 'fr',
-      'missingTranslations': true,
       'replaceSync': {
-        'Phoenix-Egi': 'Phénix-Egi',
+        'Bennu': 'bénou',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t12.js:176]: Bénou thank you.

File: ui/raidboss/data/02-arr/raid/t1.js [hunk 3]
```diff
@@ -119,4 +119,71 @@
...
+      },
+      '~effectNames': {
+      },
+    },
+    {
+      'locale': 'fr',
+      'replaceSync': {
+        'Ads': 'sphère de contrôle',
+        'Caduceus': 'Caducée',
+        'Dark Matter Slime': 'gluant de matière sombre',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t1.js:141]: Gluant de matière sombre thank you

File: ui/raidboss/data/02-arr/raid/t1.js [hunk 4]
```diff
@@ -119,4 +119,71 @@
...
+      },
+      'replaceText': {
+      },
+      '~effectNames': {
+      },
+    },
+    {
+      'locale': 'fr',
+      'replaceSync': {
+        'Ads': 'sphère de contrôle',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t1.js:139]: Sphère de contrôle thank you

File: ui/raidboss/data/02-arr/raid/t11.js [hunk 5]
```diff
@@ -251,14 +251,17 @@
...
+        'Seed Of The Sea/Rivers': 'Samen der See/Flüsse',
         'Stun': 'Betäubung',
       },
+      '~effectNames': {
+      },
     },
     {
       'locale': 'fr',
       'replaceSync': {
+        'Electric Node': 'module d\'électrochoc',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t11.js:264]: Module d\'électrochoc thank you

File: ui/raidboss/data/02-arr/raid/t5.js [hunk 6]
```diff
@@ -264,35 +265,41 @@
...
         'Unwoven Will': 'Entwobener Wille',
       },
+      '~effectNames': {
+      },
     },
     {
       'locale': 'fr',
       'replaceSync': {
         'The Right Hand of Bahamut': 'la Serre droite de Bahamut',
+        'The Scourge Of Meracydia': 'fléau de Méracydia',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t5.js:278]: Fléau de Méracydia thank you

File: ui/raidboss/data/02-arr/raid/t6.js [hunk 7]
```diff
@@ -265,12 +264,15 @@
...
         'Viscid Emission': 'Klebsporen',
       },
+      '~effectNames': {
+        'Honey-Glazed': 'Honigsüß',
+      },
     },
     {
       'locale': 'fr',
       'replaceSync': {
+        'Rafflesia': 'rafflesia',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t6.js:274]: Rafflesia 
Thank you

File: ui/raidboss/data/02-arr/raid/t1.js [hunk 8]
```diff
@@ -119,4 +119,71 @@
...
+      'replaceSync': {
+        'Ads': 'Abwehrsystem',
+        'Caduceus': 'Caduceus',
+        'Dark Matter Slime': 'Dunkelmaterien-Schleim',
+        'The Allagan megastructure': 'Allagische Superstruktur',
+      },
+      'replaceText': {
+      },
+      '~effectNames': {
+      },
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t1.js:134]: Giftpollen 
it's debuff of spuck on Caduceus

File: ui/raidboss/data/02-arr/raid/t1.js [hunk 9]
```diff
@@ -119,4 +119,71 @@
...
+      'replaceSync': {
+        'Ads': 'Sphère de contrôle',
+        'Caduceus': 'Caducée',
+        'Dark Matter Slime': 'Gluant de matière sombre',
+        'The Allagan megastructure': 'Mégastructure allagoise',
+      },
+      'replaceText': {
+      },
+      '~effectNames': {
+      },
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t1.js:147]: Poison violent
it's debuff Regorge when you are in Spit on Caduceus

File: ui/raidboss/data/02-arr/raid/t1.js (hunk 8 above)
Comment (Akurosia) [reply ui/raidboss/data/02-arr/raid/t1.js:134]: as it is not used in either the txt or js file, its not needed here. 
I already asked @quisquous and the conclusion was to only include needed elements in the replacement section

File: ui/raidboss/data/02-arr/raid/t6.js (hunk 7 above)
Comment (quisquous) [reply ui/raidboss/data/02-arr/raid/t6.js:274]: If it helps you not have to spend so much time looking at these, `replaceSync` never appears to users and is always case insensitive.  So, `Rafflesia` and `rafflesia` are functionally identical.  `replaceText` appears on timelines, so it's important to capitalize it the way you'd like it to be displayed, but `replaceSync` is only used for regular expressions.

File: ui/raidboss/data/02-arr/raid/t1.js (hunk 9 above)
Comment (Akurosia) [reply ui/raidboss/data/02-arr/raid/t1.js:147]: Same as for Giftpollen, its not used in the js/txt files so it does not need to be included,

File: ui/raidboss/data/02-arr/raid/t10.js [hunk 10]
```diff
@@ -126,6 +128,8 @@
         'Spike Flail': 'Fléau à pointes',
         'Wild Charge': 'Charge sauvage',
       },
+      '~effectNames': {
+      },
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t10.js:132]: Erhöhte Verwundbarkeit
Erratischer Puls
Markiert
Stromschlag

File: ui/raidboss/data/02-arr/raid/t10.js [hunk 11]
```diff
@@ -106,6 +106,8 @@
         'Spike Flail': 'Dornendresche',
         'Wild Charge': 'Wilde Rage',
       },
+      '~effectNames': {
+      },
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t10.js:110]: Erhöhte Verwundbarkeit
Erratischer Puls
Markiert
Stromschlag

Comment (Akurosia) [reply ui/raidboss/data/02-arr/raid/t10.js:110]: again, they are not used in either txt/js file so no need to include them.

File: ui/raidboss/data/02-arr/raid/t10.js (hunk 10 above)
Comment (Akurosia) [reply ui/raidboss/data/02-arr/raid/t10.js:132]: again, they are not used in either txt/js file so no need to include them.

File: ui/raidboss/data/02-arr/raid/t1.js (hunk 8 above)
Comment (VanSlanzar) [reply ui/raidboss/data/02-arr/raid/t1.js:134]: Ok, sorry. Same for other.

Comment (Akurosia) [reply ui/raidboss/data/02-arr/raid/t1.js:134]: dont worry :)

File: ui/raidboss/data/02-arr/raid/t7.js [hunk 12]
```diff
@@ -247,16 +249,23 @@
...
+      },
     },
     {
       'locale': 'fr',
       'replaceSync': {
         'Bioweapon Storage': 'l\'entrepôt d\'armes biologiques',
-        'Lamia Prosector': 'Lamia dissectrice',
+        'Cursed Shriek': 'Cri maudit',
+        'Cursed Voice': 'Voix maudite',
+        'Lamia Prosector': 'lamia dissectrice',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t7.js:263]: Lamia dissectrice

File: ui/raidboss/data/02-arr/raid/t7.js [hunk 13]
```diff
@@ -247,16 +249,23 @@
...
         'Melusine': 'Mélusine',
+        'Proto-Chimera': 'protochimère',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t7.js:265]: Protochimère

File: ui/raidboss/data/02-arr/raid/t8.js [hunk 14]
```diff
@@ -166,12 +166,17 @@
...
     },
     {
       'locale': 'fr',
       'replaceSync': {
         'Allagan Field': 'Champ allagois',
-        'The Avatar': 'Bio-tréant',
+        'Allagan Mine': 'mine allagoise',
+        'Clockwork Dreadnaught': 'cuirassé Dreadnaught',
+        'Landmines have been scattered': 'Des mines ont été répandues',
+        'The Avatar': 'bio-tréant',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t8.js:179]: Cuirassé Dreadnaught
Bio-tréant

File: ui/raidboss/data/02-arr/raid/t9.js [hunk 15]
```diff
@@ -492,38 +496,48 @@
         'Dalamud Dive': 'Chute de Dalamud',
         'Divebomb': 'Bombe plongeante',
         'Fireball': 'Boule de feu',
-        'Ghost Add': 'Add Fantôme',
+        'Ghost': 'fantôme',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t9.js:499]: Add Fantôme 
thank you

File: ui/raidboss/data/02-arr/raid/t9.js [hunk 16]
```diff
@@ -455,35 +453,41 @@
...
       'replaceSync': {
-        'Astral Debris': 'Débris astral',
-        'Dalamud Fragment': 'Débris de Dalamud',
-        'Firehorn': 'Corne-de-feu',
-        'Iceclaw': 'Griffe-de-glace',
+        'Astral Debris': 'débris astral',
+        'Dalamud Fragment': 'débris de Dalamud',
+        'Dalamud Spawn': 'golem de Dalamud',
+        'Firehorn': 'corne-de-feu',
+        'Iceclaw': 'griffe-de-glace',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t9.js:481]: Débris astral
Débris de Dalamud
Corne-de-feu
Griffe-de-glace

File: ui/raidboss/data/02-arr/raid/t9.js [hunk 17]
```diff
@@ -455,35 +453,41 @@
...
         'Nael Geminus': 'Nael Geminus',
         'Nael deus Darnus': 'Nael deus Darnus',
         'Ragnarok': 'Ragnarok',
-        'Thunderwing': 'Aile-de-foudre',
-        'Umbral Debris': 'Débris ombral',
+        'The Ghost Of Meracydia': 'fantôme méracydien',
+        'Thunderwing': 'aile-de-foudre',
+        'Umbral Debris': 'débris ombral',
```
Comment (VanSlanzar) [inline ui/raidboss/data/02-arr/raid/t9.js:487]: Fantôme Méracydien
Aile-de-foudre
Débris ombral

File: ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.js [hunk 18]
```diff
@@ -191,21 +188,23 @@
...
+      '~effectNames': {
+      },
     },
     {
       'locale': 'fr',
       'replaceSync': {
         'Analysis and Proving': 'Analyse et Essai',
         'Ascian Prime': 'Primo-Ascien',
-        'Blizzardsphere': 'Sphère de glace',
+        'Blizzardsphere': 'sphère de glace',
```
Comment (VanSlanzar) [inline ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.js:199]: Sphère de glace

File: ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.js [hunk 19]
```diff
@@ -191,21 +188,23 @@
...
+        'Chaosphere': 'Sphère de chaos',
         'Evaluation and Authentication': 'Évaluation et Authentification',
-        'Firesphere': 'Sphère de feu',
+        'Firesphere': 'sphère de feu',
```
Comment (VanSlanzar) [inline ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.js:202]: Sphère de feu

File: ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.js [hunk 20]
```diff
@@ -191,21 +188,23 @@
...
         'Harmachis': 'Horemakhet',
         'Igeyorhm': 'Igeyorhm',
         'Lahabrea': 'Lahabrea',
-        'Magitek Turret I': 'Tourelle magitek TM-I',
-        'Magitek Turret II': 'Tourelle magitek TM-II',
-        'Neurolink Nacelle': 'Nacelle neurolien',
+        'Magitek Turret I(?!I)': 'tourelle magitek TM-I',
+        'Magitek Turret II': 'tourelle magitek TM-II',
```
Comment (VanSlanzar) [inline ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.js:207]: Tourelle magitek TM-I
Tourelle magitek TM-II

**Comment by VanSlanzar** [comment]
I would not ask more because for the timelines, I do the content to see if the display is correct. I have not yet had time to redo the content of HW, I would make requests when I redo the content.
@Akurosia Thanks for the changes I requested

**Comment by Akurosia** [comment]
`python util\check_translation.py -f t1.js -t fr` @VanSlanzar  would be for t1

**Comment by VanSlanzar** [comment]
that tells me that, sorry for the inconvenience.
check

**Comment by Akurosia** [comment]
@VanSlanzar ok t1 was a bad example as it does not have a timeline file available.
Try to use `python util\check_translation.py -f a8s.js -t fr`

This will only work for files that have following in the js file `timelineFile:`

**Comment by VanSlanzar** [comment]
It's ok. Just is / no \
yes
Thank you much. It will save me time.

File: ui/raidboss/data/02-arr/raid/t1.js [hunk 21]
```diff
@@ -119,4 +119,71 @@
...
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Ads': 'Abwehrsystem',
+        'Caduceus': 'Caduceus',
+        'Dark Matter Slime': 'Dunkelmaterien-Schleim',
+        'The Allagan megastructure': 'Allagische Superstruktur',
+      },
+      'replaceText': {
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/raid/t1.js:131]: I don't think it's worth adding all these empty unused sections.  <_<

File: ui/raidboss/data/03-hw/pvp/shatter.js [hunk 22]
```diff
@@ -56,4 +56,64 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'missingTranslations': true,
+      'replaceSync': {
+        'The icebound tomelith A1 activates and begins to emit heat': 'Vereister Echolith A1 activates and begins to emit heat',
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/pvp/shatter.js:64]: Are these actually correct translations? They've always looked suspicious to me.

File: ui/raidboss/data/03-hw/raid/a1s.js [hunk 23]
```diff
@@ -158,8 +158,8 @@
         'Faust': 'Faust',
         'Hangar 8': 'Lagerhalle 8',
         'Machinery Bay 44': 'Kampfmaschinen-Baracke 44',
-        'Oppressor': 'Unterdrücker',
-        'Oppressor 0.5': 'Unterdrücker 0,5',
+        'Oppressor 0\\.5': 'Unterdrücker 0,5',
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/raid/a1s.js:161]: Huh, why did the tests not complain about this before...

File: ui/raidboss/data/03-hw/raid/a1s.txt [hunk 24]
```diff
@@ -122,5 +122,5 @@ hideall "--sync--"
 
 ## Enrage
 3000.0 "--sync--" sync /:E49:Oppressor/ window 3000,0
-3000.0 "--sync--" sync /:E49:Oppressor 0\.5/ window 3000,0
-3010.0 "Self-Destruct Enrage" sync /:Oppressor:E49:/
\ No newline at end of file
+3000.0 "--sync--" sync /:E49:Oppressor 0.5/ window 3000,0
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/raid/a1s.txt:125]: I think this is an incorrect change.  The previous was correct, I think?

Comment (Akurosia) [reply ui/raidboss/data/03-hw/raid/a1s.txt:125]: this is the only line, which oncluded the slash. All others were without it so then all other lines were wrong.
Also when including the slash, the find_missing_translation script reports following: `.\ui\raidboss\data\03-hw\raid\a1s.txt:125 sync ":E49:Oppressor 0\.5"`

File: ui/raidboss/data/03-hw/pvp/shatter.js (hunk 22 above)
Comment (Akurosia) [reply ui/raidboss/data/03-hw/pvp/shatter.js:64]: i will need to run PvP for it to double check (all my logfiles dont contain the line when they spawn)

File: ui/raidboss/data/02-arr/raid/t1.js (hunk 21 above)
Comment (Akurosia) [reply ui/raidboss/data/02-arr/raid/t1.js:131]: than i will remove them, if it makes you happy :)

File: ui/raidboss/data/03-hw/raid/a1s.txt (hunk 24 above)
Comment (Akurosia) [reply ui/raidboss/data/03-hw/raid/a1s.txt:125]: ok we really need to clarify how to fix this, as this also happends now on a11s as well (with the E.D.D)

Comment (quisquous) [reply ui/raidboss/data/03-hw/raid/a1s.txt:125]: I do not see this.  When I run this with and without your pull request applied, `python util/find_missing_translations.py -l de -f a1s` does not show any errors in a1s.txt, only a1s.js.

**Comment by Akurosia** [comment]
ok due to a messed up merge/rebase i will close this PR and start a fresh one (maybe skipping the changes to a1s)

**Pull request closed without merging**